autocertkit/utils.py

```diff
@@ -67,6 +67,9 @@
 REQ_CAP = "REQ"
 MULTICAST_CAP = "MULTICAST"
 
+# XCP minimum version with SR-IOV support
+XCP_MIN_VER_WITH_SRIOV = "2.6.0"
```

Comment:
Shouldn't the test class  be able to specify this restriction?

Reply:
Already limited
```
class InterHostSRIOVTestClass(IperfTestClass):
    """Iperf test between VF (in VM1 on master) and VIF (in VM2 on slave)"""

    REQUIRED_FOR = ">= %s" % XCP_MIN_VER_WITH_SRIOV
```